docs/plugins.md

```diff
@@ -14,183 +14,154 @@ Looking for help using Snowpack in your project?
 
 ## Overview
 
-Snowpack runs through 3 stages internally: _build_, _transform_, and _bundle_. Snowpack plugins hook into **one** of these stages:
+There are two types of Snowpack plugins: **build plugins** and **bundler plugins.** Unless you are making a plugin specifically for a bundler like Rollup, Parcel, or webpack, you’re probably making a build plugin.
```

Comment:
The actual docs changes. I won’t comment here; please review the Vercel link and leave feedback!